kube-request-access.go

```diff
@@ -88,6 +90,7 @@ func main() {
 
 	cmd.Flags().StringVar(&cfg.grantedRoleName, "granted-role-name", "", "Name of the role that is given to a user temporarily when a request is granted")
 	cmd.Flags().StringSliceVar(&cfg.alwaysAllowedGroupNames, "always-allowed-group-name", nil, "Name of a group whose members will be allowed to execute commands without a request and grant")
+	cmd.Flags().StringSliceVar(&cfg.alwaysAllowedUserNames, "always-allowed-user-name", nil, "Unique user name of a user who will be allowed to execute commands without a request and grant")
```

Comment:
I think the "Unique" here is superfluous, as user names in Kubernetes should always be unique.